packages/contracts-bedrock/scripts/deploy/ManageDependencies.s.sol

```diff
@@ -47,17 +47,18 @@ contract ManageDependenciesInput is BaseDeployIO {
 }
 
 contract ManageDependencies is Script {
+    // TODO: Fix dependency management since SystemConfig does not handle them anymore
```

Comment:
Missing TODO?

Reply:
The thing is that `ManageDependencies` is a new script that now it doesn't have any effect, so we have to know what to do with it before deleting it